src/main.jsx

```diff
@@ -8,3 +8,4 @@ ReactDOM.createRoot(document.getElementById("root")).render(
     <App />
   </React.StrictMode>
```

Comment:
telling from this line it is visible that you have imported react
I suggest you only import the strict mode since it is of a good help to your warning and errors awareness  
```suggestion
import { StrictMode } from "react"; 
```

Then for the place where you are calling your React.StrictMode component I suggest the below: 
```suggestion
 <StrictMode>
    <App />
  </StrictMode>
```